### PR TITLE
Create Lavishmunjal.json

### PR DESCRIPTION
**Pull request by Lavishmunjal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4389"></a>

### CODE REVIEW

File: data/Lavishmunjal.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "type": "personal",
+  "bio": "Computer Science Student || Learning Java + DSA || Learning Web Development || Bachelor of Technology (BTech) || Tech Enthusiast",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Lavishmunjal8",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkdIn",
```
Comment (rupali-codes) [inline data/Lavishmunjal.json:13]:
```suggestion
      "name": "Follow me on LinkedIn",
```

File: data/Lavishmunjal.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "url": "https://www.linkedin.com/in/lavishmunjal/",
+      "icon": "FaLinkdin"
```
Comment (rupali-codes) [inline data/Lavishmunjal.json:15]:
```suggestion
      "icon": "FaLinkedin"
```

**Review by rupali-codes — COMMENTED** [review]
rest looks great to me :)

File: data/Lavishmunjal.json (hunk 2 above)
Comment (Lavishmunjal) [reply data/Lavishmunjal.json:15]: Hello, where i found my linkfree link??

Comment (Panquesito7) [reply data/Lavishmunjal.json:15]: > Hello, where i found my linkfree link??

Your profile will be visible and added once the PR is merged. 🙂

**Pull request merged**